natives.json

```diff
@@ -102240,111 +102240,111 @@
 	},
 	"_NAMESPACE71": {
 		"0x049D5C615BD38BAD": {
-			"name": "_0x049D5C615BD38BAD",
-			"comment": "",
+			"name": "_SHOW_TOOLTIP",
+			"comment": "Example : https://pastebin.com/GvdBp8Dh",
 			"params": [
 				{
-					"type": "Any",
+					"type": "Any*",
 					"name": "p0"
 				},
 				{
-					"type": "Any",
+					"type": "Any*",
 					"name": "p1"
 				},
 				{
-					"type": "Any",
+					"type": "int",
 					"name": "p2"
 				}
 			],
-			"return_type": "Any",
+			"return_type": "void",
```

Comment:
Please _do not_ ever change something to ``void``. Please change this back to ``Any``.